src/js/LottieView.js

```diff
@@ -120,7 +120,7 @@ class LottieView extends React.Component {
       android: () =>
         UIManager.dispatchViewManagerCommand(
           handle,
-          UIManager.LottieAnimationView.Commands[name],
+          UIManager.getViewManagerConfig('LottieAnimationView').Commands[name],
```

Comment:
Would this break support of RN < 0.58 since `getViewManagerConfig` doesn't exist in older versions or am I missing something?

Reply:
@XeroX Agree,  something like [this](https://github.com/lelandrichardson/react-native-safe-module/pull/6/files#diff-b3efefcfc3c27304dfc5617d511ce810R21) would be helpful

Reply:
@andrewtremblay can you create a fix for this and ping me once it's ready?

Reply:
@emilioicai: Just pushed a commit that should address the backwards compatibility issues.

Thanks everyone for your patience!

Reply:
@emilioicai: Just pushed a commit that should address the backwards compatibility issues.

Thanks everyone for your patience!